blackboxopt/utils.py

```diff
@@ -99,3 +102,35 @@ def sort_evaluations(evaluations: Iterable[Evaluation]) -> Iterable[Evaluation]:
             )
         ).hexdigest(),
     )
+
+
+def save_study_as_pickle(
+    search_space: ps.ParameterSpace,
+    objectives: List[Objective],
+    evaluations: List[Evaluation],
+    pickle_file_path: os.PathLike,
+    overwrite: bool = False,
+):
+    """Save space, objectives and evaluations as pickle at `pickle_file_path`."""
+    if Path(pickle_file_path).exists() and not overwrite:
+        raise ValueError(f"{pickle_file_path} exists and overwrite is False")
+
+    with open(pickle_file_path, "wb") as fh:
```

Comment:
Not a crucial point, but I believe this to fail if the directory of `pickle_file_path` does not exist. Should it be created in that case or fail with a meaningful message?

Reply:
I went with raising a helpful error and leaving the creation to the user. Thanks!